CHANGELOG_UNRELEASED.md

```diff
@@ -177,6 +177,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - in `matrix.v`, `card_matrix` -> `card_mx`
 
+- in `finset`
+  + `mem_imset_eq`  -> `mem_imset`
+  + `mem_imset2_eq` -> `mem_imset2`
```

Comment:
I think this is a changelog error introduced by the rebase.

Reply:
my bad, should be fixed